<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClient</class>
 <widget class="QDialog" name="CreateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>51</x>
         <y>43</y>
         <width>541</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Personal Details</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <property name="spacing">
             <number>35</number>
            </property>
            <item>
             <widget class="QLabel" name="label_name_2">
              <property name="text">
               <string>Last Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>lineEdit_lname</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_lname_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <property name="spacing">
             <number>35</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>First Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_fname_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <property name="spacing">
             <number>56</number>
            </property>
            <item>
             <widget class="QLabel" name="label_address_2">
              <property name="text">
               <string>Address</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>lineEdit_address</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_address_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <property name="spacing">
             <number>16</number>
            </property>
            <item>
             <widget class="QLabel" name="label_dob_2">
              <property name="text">
               <string>Date Of Birth</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_dob</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_dob_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="spacing">
             <number>33</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <property name="spacing">
               <number>60</number>
              </property>
              <item>
               <widget class="QLabel" name="label_gen_2">
                <property name="text">
                 <string>Gender</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_gender</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_gender_2"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_nationality_2">
              <property name="text">
               <string>Nationality</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_nationality</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_nationality_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>310</y>
         <width>146</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_prDetails">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>441</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string>Bank Account</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <property name="spacing">
             <number>47</number>
            </property>
            <item>
             <widget class="QLabel" name="label_acct_2">
              <property name="text">
               <string>Account Type</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_acct</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_acct_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <property name="spacing">
             <number>70</number>
            </property>
            <item>
             <widget class="QLabel" name="label_caste_2">
              <property name="text">
               <string>User_Type</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_usrt</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_usrt_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <property name="spacing">
             <number>68</number>
            </property>
            <item>
             <widget class="QLabel" name="label_mob_2">
              <property name="text">
               <string>Mobile No.</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_mob</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_mob_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <property name="spacing">
             <number>85</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Email ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_email_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_bnAcc_b">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>400</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_bnAcc_n">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>400</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>451</width>
         <height>311</height>
        </rect>
       </property>
       <property name="title">
        <string>Security</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26">
            <property name="spacing">
             <number>13</number>
            </property>
            <item>
             <widget class="QLabel" name="label_sq_2">
              <property name="text">
               <string>Security Question</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_sq</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_sq_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Answer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ans"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_usrname_crcl">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_username_crcl"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_password_crcl">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_password_crcl"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_sec_create">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>380</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sec_b">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>380</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
